src/Morfologik.TestFramework/TypeExtensions.cs

```diff
@@ -11,7 +11,7 @@ public static class TypeExtensions
         /// </summary>
         public static Stream getResourceAsStream(this Type t, string name)
         {
-            return t.Assembly.FindAndGetManifestResourceStream(t, name);
+            return t.Assembly.FindAndGetManifestResourceStream(t, name)!;
```

Comment:
Perhaps it would be better here to coalesce this with a more meaningful exception than returning null, i.e. `?? throw new InvalidOperationException($"Could not find manifest resource stream '${name}'.");

Reply:
Good idea. I added an exception.